Classes/Constants.php

```diff
@@ -0,0 +1,6 @@
+<?php
+if (!defined('Tx_Mklib_Domain_Model_Iso_Iban')) {
+    define('Tx_Mklib_Domain_Model_Iso_Iban', 'Tx_Mklib_Domain_Model_Iso_Iban');
```

Comment:
It seems that you didn't remove the code where Tx_Mklib_Domain_Model_Iso_Iban and Tx_Mklib_Domain_Model_Iso_SwiftBic were defined previously. Please make sure to move (not copy) all constants to that file